cpp/include/cuspatial/polygon_bounding_box.hpp

```diff
@@ -25,6 +25,8 @@ namespace cuspatial {
 /**
  * @brief Compute minimum bounding boxes for a set of polygons.
  *
+ * @ingroup spatial_relationship
```

Comment:
`spatial_relationship` isn't my favorite. It feels rather vague, but I assume you all discussed this already in your meetings and I don't immediately have a better suggestion so I'm fine getting this merged and maybe pondering future improvements.

Reply:
We haven't discussed. The names of groups was an item for discussion in this PR.

Reply:
Given the time constraints, I'd say we just leave this as is and revisit in 22.08?